core/src/main/scala/kafka/tools/PerfConfig.scala

```diff
@@ -28,9 +28,9 @@ class PerfConfig(args: Array[String]) {
     .ofType(classOf[java.lang.Long])
   val reportingIntervalOpt = parser.accepts("reporting-interval", "Interval at which to print progress info.")
```

Comment:
How about change the description to something like `Interval in ms at which to print progress info`?